src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplay.cs

```diff
@@ -37,11 +37,19 @@ public static class SymbolDisplay
             ITypeSymbol symbol,
             CodeAnalysis.NullableFlowState topLevelNullability,
             SymbolDisplayFormat format = null)
-#pragma warning restore RS0026 // Do not add multiple public overloads with optional parameters
         {
             return ToDisplayParts(symbol, topLevelNullability, format).ToDisplayString();
         }
 
+        public static string ToDisplayString(
```

Comment:
>ToDisplayString [](start = 29, length = 15)

Are there tests for the new `SymbolDisplay` methods?

Reply:
Not currently. I want to come back and refactor these, as we really need to be able to handle non `TypeSymbol` inputs. I'm just doing the simple thing for now, but forgot to put a prototype. I'll add one

---
In reply to: [269216952](https://github.com/dotnet/roslyn/pull/34429#discussion_r269216952) [](ancestors = 269216952)